sympy/physics/mechanics/tests/test_body.py

```diff
@@ -62,8 +62,22 @@ def test_particle_body():
     assert hasattr(particle_body, 'frame')
     assert hasattr(particle_body, 'masscenter')
     assert hasattr(particle_body, 'mass')
+    assert particle_body.inertia == (Dyadic(0), particle_body.masscenter)
+    assert particle_body.central_inertia == Dyadic(0)
+    assert not particle_body.is_rigidbody
+
+    particle_body.central_inertia = inertia(particle_frame, 1, 1, 1)
+    assert particle_body.central_inertia == inertia(particle_frame, 1, 1, 1)
+    assert particle_body.is_rigidbody
 
+    particle_body = Body('particle_body', mass=particle_mass)
     assert not particle_body.is_rigidbody
+    point = particle_body.masscenter.locatenew('point', particle_body.x)
+    point_inertia = particle_mass * inertia(particle_body.frame, 0, 1, 1)
+    particle_body.inertia = (point_inertia, point)
+    assert particle_body.inertia == (point_inertia, point)
+    assert particle_body.central_inertia == Dyadic(0)
```

Comment:
I'm confused about this. It seems that you set the inertia of this body to be Ixx=0, Iyy=1, Izz=1 about a point 1 unit along x from the mass center. Then you ask for the inertia about the mass center here and it gives zero. If the body has inertia about y and z, shouldn't this be nonzero? Or is it that the mass=1 and r=1, so m*r**2=1, then you substract 1 from 1 for the y and z terms. I guess that is it.

Reply:
I convinced myself there.